client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Layout, Input, Space, Row } from 'antd';
+import HeaderNavbar from '../../content/HeaderNavbar/HeaderNavbar';
+import RobotContainer from '../../content/RobotContainer/RobotContainer'
+import CreateRobotContainer from '../../content/RobotContainer/CreateRobotContainer'
+
+const { Search } = Input;
+
+/**
+ * @description Overview page, where all robots are displayed and can be opened.
+ * @category Client
+ * @component
+ */
+const RobotOverview = () => {
+  const [searchValue, setSearchValue] = useState('');
+
+  const updateSearchValue = (val) => {
+    setSearchValue(val);
+  };
+
+  const createRobotBoxes = (searchValue2) => {
```

Comment:
Please add documentation to all the functions

Reply:
✅